contracts/tokens/STFactory.sol

```diff
@@ -1,21 +1,55 @@
 pragma solidity ^0.5.0;
 
-import "./SecurityToken.sol";
+import "./SecurityTokenProxy.sol";
+import "../proxy/OwnedUpgradeabilityProxy.sol";
 import "../interfaces/ISTFactory.sol";
+import "../interfaces/ISecurityToken.sol";
+import "../interfaces/IOwnable.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "../interfaces/IModuleRegistry.sol";
+import "../interfaces/IPolymathRegistry.sol";
 import "../datastore/DataStoreFactory.sol";
 
 /**
  * @title Proxy for deploying SecurityToken instances
  */
-contract STFactory is ISTFactory {
+contract STFactory is ISTFactory, Ownable {
+
     address public transferManagerFactory;
```

Comment:
As logic contract can be upgraded, We should allow upgrading of transferManagerFactory as well?

Reply:
Done (if by upgrading you mean modifying the address).